src/OSPSuite.Core/Import/DataFormatParameter.cs

```diff
@@ -128,8 +130,19 @@ public override bool EquivalentTo(DataFormatParameter other)
 
       public override string TooltipDescription()
       {
-         
-         return Captions.Importer.MappingHint(ColumnName, MappedColumn.Name, MappedColumn.Unit.SelectedUnit);
+         if (MappedColumn.ErrorStdDev == Constants.STD_DEV_GEOMETRIC)
+            return Captions.Importer.MappingHintGeometricError;
+
+         if (!MappedColumn.Unit.ColumnName.IsNullOrEmpty())
+         {
```

Comment:
Remoe the { } if before and after you don't use them.  I don't care if we use {} or not , but the code within the same function needs to be consistent

Reply:
corrected